test/miscellaneous_tests/symbolic_stoichiometry.jl

```diff
@@ -217,9 +217,9 @@ let
         d, Y --> 0
     end
 
-    u0 = [:X => 1, :Y => 1]
-    ps_int = [:p => 1.0, :n1 => 3, :n2 => 4, :d => 0.5]
-    ps_dec = [:p => 1.0, :n1 => 0.5, :n2 => 2.5, :d => 0.5]
+    u0 = [:X => 8, :Y => 8]
+    ps_int = [:p => 1.0, :n1 => 3, :n2 => 4, :d => 0.1]
```

Comment:
```suggestion
    ps_int = (:p => 1.0, :n1 => 3, :n2 => 4, :d => 0.1)
```

Reply:
Why was this marked resolved? Is this intended to have mixed types or not, and if it is then it shouldn't use a vector.

Reply:
I accepted your suggested commit to change it to a Tuple, and after that I resolved it

Reply:
(With ModelingToolkit now automatically converting stuff to whatever type they are designated as, and mixed types being handled in the system definition, and not the input, I started using a mix of Tuple/Vector definitions, just to diversify the tests, but here you said you preferred to always make them Tuples, so I do that now)

Reply:
Weird it is still showing the old code for me.

Reply:
I think we shouldn't rely on MTK properly converting between types unless that is stated somewhere as an official feature, so should use containers that preserve types (and use such workflows in docs for our users). I don't trust that to not change in the future unless it is offiicially documented as the behavior going forward.

Reply:
makes sense